src/main/java/uk/gov/mca/beacons/service/accounts/AccountHolderController.java

```diff
@@ -24,21 +25,38 @@
 
   private final AccountHolderMapper accountHolderMapper;
 
+  private final GetAccountHolderByIdService getAccountHolderByIdService;
+
   private final GetAccountHolderByAuthIdService getAccountHolderByAuthIdService;
 
   private final CreateAccountHolderService createAccountHolderService;
 
   @Autowired
   public AccountHolderController(
     AccountHolderMapper accountHolderMapper,
+    GetAccountHolderByIdService getAccountHolderByIdService,
     GetAccountHolderByAuthIdService getAccountHolderByAuthIdService,
     CreateAccountHolderService createAccountHolderService
   ) {
     this.accountHolderMapper = accountHolderMapper;
+    this.getAccountHolderByIdService = getAccountHolderByIdService;
     this.getAccountHolderByAuthIdService = getAccountHolderByAuthIdService;
     this.createAccountHolderService = createAccountHolderService;
   }
 
+  @GetMapping(value = "/{id}")
+  public WrapperDTO<AccountHolderDTO> getAccountHolder(
+    @PathVariable("id") String id
```

Comment:
Does the Path Variable need to be a String? If it's always going to be a UUID then we could type it as that and avoid having to do the conversion later on. Also has the advantage of getting Spring to do the validation for us

Reply:
Nice!! Done 👍🏻